app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsSheetDialogFragment.kt

```diff
@@ -102,12 +104,19 @@ class QuickSettingsSheetDialogFragment : AppCompatDialogFragment() {
     }
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-        val customDialog = if (promptGravity == BOTTOM) {
-            return BottomSheetDialog(requireContext(), this.theme)
+        return if (promptGravity == BOTTOM) {
```

Comment:
I'm conflicted about the `return if` here. 

On the one hand, this is a large reasonably complicated `then` block, so treating this `if` as an expression looks like a contraction too far. @NotWoods may be tempted to get rid of the block and `return` altogether and set the 

```kotlin
override fun onCreateDialog(savedInstanceState: Bundle?) = if (…) { … } else { … }
```

On the other, I like the `if` expressions.

I think on consideration, if the `if` expression gets any more complicated, I'd prefer we should move it to an `if` statement. Others with more kotlin sense (including me in 6 months time) may disagree.

Reply:
I think it's simple enough here to leave as a large if statement. I agree that if the logic becomes more complicated we should have two `return` statements.